apps/golden-sample-app/src/environments/environment.ts

```diff
@@ -5,38 +5,20 @@
 import { HTTP_INTERCEPTORS } from '@angular/common/http';
 import { Provider } from '@angular/core';
 import { AuthConfig } from 'angular-oauth2-oidc';
-import { TransactionsInterceptor } from '../app/interceptors/transactions.interceptor';
-import { AccountsInterceptor } from '../app/interceptors/accounts-interceptor';
 import { AchPositivePayInterceptor } from '../app/interceptors/ach-positive-pay.interceptor';
 import { Environment } from './type';
-import { EntitlementsInterceptor } from '../app/interceptors/entitlements-interceptor';
 
 const mockProviders: Provider[] = [
-  {
-    provide: HTTP_INTERCEPTORS,
-    useClass: TransactionsInterceptor,
-    multi: true,
-  },
-  {
-    provide: HTTP_INTERCEPTORS,
-    useClass: AccountsInterceptor,
-    multi: true,
-  },
   {
     provide: HTTP_INTERCEPTORS,
     useClass: AchPositivePayInterceptor,
     multi: true,
   },
-  {
-    provide: HTTP_INTERCEPTORS,
-    useClass: EntitlementsInterceptor,
-    multi: true
-  }
 ];
 
 export const environment: Environment = {
   production: false,
-  apiRoot: 'https://app.dev.sdbx.live.backbaseservices.com/api',
+  apiRoot: '/api',
```

Comment:
Why this change? Are you using proxy?

Reply:
reverse-proxying the apis is needed for user-context cookie and file downloads 😞